src/kixi/heimdall/components/metrics.clj

```diff
@@ -11,6 +11,61 @@
             [metrics.ring.expose :as expose]
             [taoensso.timbre :as log]))
 
+(def upper-name
+  (memoize
```

Comment:
why the memoize here and on `uri-method-stats->metric-name`?

Reply:
not sure what you are getting at they are different functions being memoized...

Reply:
ok